package.json

```diff
@@ -7,6 +7,7 @@
   "type": "module",
   "scripts": {
     "build": "expo-module build",
+    "postbuild": "sh postbuild.sh",
```

Comment:
did you tried the `postinstall` ([link](https://yarnpkg.com/advanced/lifecycle-scripts#postinstall))

If it works, you won't need to update Makefile because it will run automatically after `yarn` or `yarn install` command.

Reply:
Finally putting the code in the prepare stage is better: https://docs.npmjs.com/cli/v10/using-npm/scripts#life-cycle-scripts